bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/keys/show/ShowKeysUI.java

```diff
@@ -223,7 +223,7 @@ protected void configureShell(Shell newShell) {
 
 			Color color = JFaceResources.getColorRegistry().get(POPUP_COLOR_BG);
 			newShell.setBackground(color);
-			newShell.setAlpha(170);
+			newShell.setAlpha(100);
```

Comment:
How about 130?
```suggestion
			newShell.setAlpha(130);
```

Reply:
Thanks @BeckerWdf. I have made the changes. Please have a look